R/io-ipc-functions.R

```diff
@@ -54,19 +44,21 @@ pl__scan_ipc <- function(
     row_index_name = NULL,
     row_index_offset = 0L,
     storage_options = NULL,
-    memory_map = TRUE,
```

Comment:
Why remove this argument? I don't see any mentions of deprecation in the docs

Reply:
This does not actually do anything.
That is probably a bug in the Python documentation, which has already been removed from the Rust side.
See pola-rs/polars#18622

Reply:
Alright, thanks